scripts/client/_otter_env_core.sh

```diff
@@ -120,7 +120,7 @@ then
 fi
 : ${otter_lib_perl5:=$OTTER_HOME/lib/perl5}
 
-PERL5LIB="\
```

Comment:
I would remove line 191 and do the export here (or move this to line 191 where the export actually happens).
Much easier to maintain having it together and not in different locations.

Also why `${PERL5LIB:+${PERL5LIB}:}`?
I would just do `$PERL5LIB:`

Reply:
Because in the latter case, if PERL5LIB has been unset or empty you end up with something along the lines ':/foo/bar:/baz', which depending on the specific implementation of Bourne-shell logic may or may not mean to the current shell "search for modules _in the current directory_ first". This is both a security risk and potentially confusing to users.

Reply:
alright, good to know that.
still, doing the export at the same time sounds like a good idea to me (but not really a big issue)...

Reply:
I would still leave it where it is, _i.e._ after we are really, really done messing with the value of PERL5LIB. Technically speaking the "exported" status is assigned to the variable name, meaning that subsequent changes to the value of an exported variable still get propagated to subprocesses - but at least in my own sysadmin experience, there seems to be an unwritten convention that you only export when done setting the value.